charts/app-config/values-integration.yaml

```diff
@@ -1415,6 +1415,21 @@ govukApplications:
             secretKeyRef:
               name: signon-app-chat
               key: oauth_secret
+        - name: SMART_SURVEY_URL
+          valueFrom:
+            secretKeyRef:
+              name: govuk-chat-smart-survey
+              key: url
+        - name: SMART_SURVEY_BASIC_AUTH_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: govuk-chat-smart-survey
+              key: username
+        - name: SMART_SURVEY_BASIC_AUTH_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: govuk-chat-smart-survey
+              key: password
```

Comment:
The terminology of these seem a little different to what Smart Survey uses of API Key and API Key secret - even if they are ended up used as basic auth - so might be better to use their terminology.

Reply:
They sort of half use the terminology. I ended up using the username and pw due to potential confusion around stuff like this

<img width="584" alt="image" src="https://github.com/user-attachments/assets/832124ac-78f8-4039-a5b7-1f8940b2b58f">

Reply:
Ah I can understand that, though I assume they had to use that there because of the library they're using for modelling those API calls.

They also manage to go further inconsistent in their attempt to make it clear: https://docs.smartsurvey.io/docs/getting-started#getting-your-api-key:~:text=Basic%20Auth%2C%20Usernames,the%20Token%20Secret. by calling them token and token secret. I think API Key and API Key secret is the easiest to match to the credentials view though.

Reply:
okay cool i'll update them in the secrets manager and update them here 👍